### DISCUSSION TITLE
How to regenerate a @unique value by Database?

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#7591] [Q&A] [answered]
Hello!

I don't know if it is possible by design but I have this schema:

```prisma
model User {
  id              String   @id @default(cuid())
  firstName       String
  lastName        String
  email           String   @unique
  apiKey          String   @unique @default(cuid())

  createdAt DateTime @default(now())
}
```

I would regenerate the column "apiKey". I could use UUID and prisma.entity.update method but I would prefer Prisma to take care of it for me OR database.

What I am looking for?
Something like it:

```ts
import { Prisma } from '@prisma/client'

prisma.user.update(
   data: {
         apiKey: Prisma.DEFAULT,
   }
    where: { id },
)
```

Is it possible?

**Comment by ryands17** [comment, marked as answer]
@felinto-dev 👋 

Currently this isn't possible, you would need to specify your own UUID or use a library for that. I would suggest opening a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this :)

**Reply by felinto-dev** [reply]
well done! =D

https://github.com/prisma/prisma/issues/7603

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
